addons/l10n_in/tests/test_hsn_summary.py

```diff
@@ -57,7 +57,7 @@ def _assert_sub_test(self, test, results):
         else:
             super()._assert_sub_test(test, results)
 
-    def create_base_line_dict(self, l10n_in_hsn_code, quantity, price_unit, uom, taxes=None, product=None):
+    def create_base_line_dict(self, l10n_in_hsn_code, quantity, price_unit, discount, uom, taxes=None, product=None):
```

Comment:
couldn't you just call the method with the price_unit with discount applied? and not change this method at all?